rpa-bot/1.3/modules/ROOT/pages/configuration.adoc

```diff
@@ -16,29 +16,35 @@ After saving your changes, restart the RPA Bot Service for the changes to take e
 [[common-settings]]
 == Common Settings
 
-image:bot-configurator-common.png[The RPA Bot configurator application showing the Common settings panel, 50%, 50%]
+image:bot-configurator-common.png[The RPA Bot Configurator application showing the Common settings panel, 50%, 50%]
 
 * *RPA Bot ID*
 +
 ID generated during the installation process of RPA Bot, which identifies the bot in RPA Manager (even if the IP or MAC-Address changes). If you need a new ID, uninstall and then reinstall RPA Bot.
 * *RPA Manager to connect*
 +
 The URL address of the RPA Manager instance the bot connects to. There is no default setting.
-* *Use HTTPS as protocol layer*
-+
-Specifies whether the bot connects with RPA Manager via HTTP or HTTPS.
 * *API Key*
 +
 The API key entered during the installation of the robot. You can change this setting.
+* *TEST CONNECTION*
++
+Click *TEST CONNECTION* to verify that the bot can establish a connection to RPA Manager.
 * *Alive interval*
 +
 Time interval in milliseconds after which the bot connects to the RPA Manager to show that it is still alive.
 * *Check interval*
 +
 Time interval in milliseconds after which the bot connects to RPA Manager and checks for provided packages to collect (such as scheduling tasks or activities).
-* *Cleanup workflows on start*
+* *Cleanup activities on start*
```

Comment:
Suggest to the UI designer to use "Clean up" (verb form) :)

Reply:
There is a bug for this already 
@tasmoore [(Bot) Bot Configurator - UI CX/UX Bugs](https://gus.lightning.force.com/a07EE00001cJLQOYA4)